tests/BaGet.Core.Tests/Services/FileStorageServiceTests.cs

```diff
@@ -191,6 +234,25 @@ protected async Task<string> ToStringAsync(Stream input)
                     return await reader.ReadToEndAsync();
                 }
             }
+
+            public IEnumerable<string> OutsideStorePathData
+            {
+                get
+                {
+                    string fullPath = Path.GetFullPath(_storePath);
+                    yield return "../file";
+                    yield return ".";
+                    yield return $"../{Path.GetFileName(_storePath)}";
+                    yield return $"../{Path.GetFileName(_storePath)}suffix";
+                    yield return $"../{Path.GetFileName(_storePath)}suffix/file";
+                    yield return fullPath;
+                    yield return fullPath + Path.DirectorySeparatorChar;
```

Comment:
This test case fails on Windows. See my comment on line 109 of `FileStorageService`.

Reply:
I'm loving how thorough these tests are 👍